content/800-data-platform/200-projects/100-add-project/110-create-new-project-from-template.mdx

```diff
@@ -0,0 +1,107 @@
+        * **SaaS**. Schema for an identity service with a SaaS with a REST API.
+        * **URL Shortener**. Schema for a URL shortener with a REST API.
+        * **Empty**. An empty schema with no models, to build your own schema.
+    2. (Optional) Clear **Include sample data** if you do not want to seed the database with sample data.
+
+        <img
+            src="https://res.cloudinary.com/prismaio/image/upload/v1666649988/docs/pdp/create-manage-projects/create-project-02-clear-include-sample-data_qvdps1.png"
+            alt="Prisma Data Platform - Create a new project - Select a template project and seed your database"
+            width="750px"
+        />
+    3. Click **Next**.
+4. On the _Configure environment_ page, connect your database and configure the Data Proxy location.
+    1. In **Connection string**, paste your database connection string.
+        <Admonition type="alert">
+
+        **Important**<br /><br />
+        If you provide a connection string to a non-empty database (with existing tables and data), the project creation fails.<br /><br />
+
+        To add a project with an existing database, see [Import an existing project with an existing database](/data-platform/projects/add-project/import-existing-project-existing-db).
+        </Admonition>
+    2. Under _Data Proxy_, from the **Location** drop-down menu, select the geographic location for the Data Proxy.
+        <Admonition>
+        
+        **Note**<br /><br />
+        Select a Data Proxy location that is close to the geographic location of your database.<br /><br />
+        If you select a Data Proxy location that is remote from your database, then when your application sends requests to your database through the Data Proxy, the responses might be slow or time out due to higher latency.
+        </Admonition>
+    3. From **Static IPs**, select **Enabled** if your database is behind a firewall and you can only configure external access from specific IP addresses. Copy the IP addresses and add them to the allowlist of your database.
+        <Admonition>
+
+        **Note**<br /><br />
+        For more information, see [Static egress IP address](/data-platform/static-ips). 
+        </Admonition>
+    4. Click **Next**.
+        <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1666700046/docs/pdp/create-manage-projects/create-project-03-configure-database-connection-and-data-proxy_itu24z.png"
+        alt="Prisma Data Platform - Create a new project - Configure the database connection string and select a Data Proxy location"
+        width="750px"
+        />
+5. On the _Deploy project_ page, you can deploy your project to Vercel, and save your connection strings and environment variables.
+6. Click **Done**.
+        
+## Result
+
+Based on the application template you selected, the Data Browser opens with a list of the models (tables or collections) in the database.
+
```

Comment:
Actually, when finished, user is redirected to `Query console`. We might wanna change the copy?